fl_training.py

```diff
@@ -107,6 +107,7 @@ def compute_test_score_with_scenario(scenario, is_save_fig=False):
         scenario.x_test,
         scenario.y_test,
         scenario.is_early_stopping,
+        scenario.amounts_per_node,
```

Comment:
At this point, I would pass the whole scenario object to the `compute_test_score_with_scenario()` method

Reply:
You mean to `compute_test_score()`?

Reply:
Yes you are right, I meant `compute_test_score()`

Reply:
We can't do that simply. Because we need a `compute_test_score()` independent from scenarios, to be used in contributivity measure algorithms when needing to launch federated learning on an arbitrary coalition.

Reply:
Ah yes you are right !